src/Queue/Queue.php

```diff
@@ -13,32 +13,32 @@ final class Queue implements IQueue
 	private IConnection $connection;
 
 	/**
-	 * @var bool
+	 * @ var bool
 	 */
 	/*private $passive;*/
 
 	/**
-	 * @var bool
+	 * @ var bool
 	 */
 	/*private $durable;*/
 
 	/**
-	 * @var bool
+	 * @ var bool
 	 */
 	/*private $exclusive;*/
 
 	/**
-	 * @var bool
+	 * @ var bool
 	 */
 	/*private $autoDelete;*/
 
 	/**
-	 * @var bool
+	 * @ var bool
 	 */
 	/*private $noWait;*/
 
 	/**
-	 * @var array
+	 * @ var array
```

Comment:
This seems weird. Why is there a space?

Reply:
PHPStan means that is iregular PHP code and it reports error.

Reply:
This is a normal PHP annotation https://php-annotations.readthedocs.io/en/stable/UsingAnnotations.html

Reply:
But code is commented.

Reply:
I see. God knows why. Give me a minute.

Reply:
I know why. Queue properties get declared separately. Could you, please, remove all this dead code? 🙌

Reply:
OK

Reply:
Done